### PR TITLE
Pump auto docs

### PR DESCRIPTION
**Pull request by zalbiraw**

### CODE REVIEW

**Comment by zalbiraw** [comment]
WIP

File: tyk-docs/content/shared/pump-config.md [hunk 1]
```diff
@@ -1,4 +1,1270 @@
...
+```
+
+### analytics_storage_type
+EV: <b>TYK_PMP_ANALYTICSSTORAGETYPE</b><br />
+Type: `string`<br />
+
+Sets the analytics storage type. Where the pump will be fetching data from. Currently, only
+the `redis` option is supported.
+
+### analytics_storage_config
```
Comment (sedkis) [inline tyk-docs/content/shared/pump-config.md:3559]: I don't think these sub fields are documented, are they?

**Review by sedkis — CHANGES_REQUESTED** [review]

File: tyk-docs/content/shared/pump-config.md (hunk 1 above)
Comment (zalbiraw) [reply tyk-docs/content/shared/pump-config.md:2932]: Are you saying they should not be documented?

**Review by sedkis — APPROVED** [review]
LGTM

**Comment by buger** [comment]
@marksou note that merge to 3.2 brought some SQL changes to dashboard too

**Pull request merged**